src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/reporting/generator/BookingsReportGeneratorTest.kt

```diff
@@ -852,6 +853,55 @@ class BookingsReportGeneratorTest {
           .produce(),
       )
       .withDateOfBirth(LocalDate.parse("1571-12-27"))
+      .withPnc(pnc)
+      .produce()
+
+    val actual = reportGenerator.createReport(
+      listOf(booking).toBookingsReportData { crn ->
+        PersonSummaryInfoResult.Success.Full(crn, caseSummary)
+      },
+      BookingsReportProperties(ServiceName.temporaryAccommodation, null, 2023, 4),
+    )
+    assertThat(actual.count()).isEqualTo(1)
+    assertThat(actual[0][BookingsReportRow::personName]).isEqualTo("Johannes Kepler")
+    assertThat(actual[0][BookingsReportRow::pncNumber]).isEqualTo(pnc)
+    assertThat(actual[0][BookingsReportRow::gender]).isEqualTo("Male")
+    assertThat(actual[0][BookingsReportRow::ethnicity]).isEqualTo("Other White")
+    assertThat(actual[0][BookingsReportRow::dateOfBirth]).isEqualTo("1571-12-27")
+  }
+
+  @Test
+  fun `The personal details columns with empty pnc returned when pnc not returned from offerder service`() {
```

Comment:
I think this test is unnecessary as this check is already done in `The personal details columns are empty if there is no person information available for the booking`.

Reply:
Make sense to remove it as its already covered.
I initially added this testcase because pnc is optional field in the response